docs-source/getting-started/building-docs.rst

```diff
@@ -0,0 +1,51 @@
+Building the documentation
+==========================
+
+This documentation is written as reStructuredText and built using
```

Comment:
We should probably make clear here that this step can be skipped if you just want to get started.

Reply:
Good point, done